src/services/SecurutyService.ts

```diff
@@ -1,16 +1,49 @@
+/* eslint-disable class-methods-use-this */
 import { UserInfo } from "../app/features/auth/models";
 
 const REFRESH_TOKEN = "refreshToken";
 const ACCESS_TOKEN = "accessToken";
 const USER_INFO = "userInfo";
 
+export class SecurityError extends Error {
```

Comment:
is there a reason for this?

Reply:
To be able to detect security errors (like expired tokens) in actions and sign user out.